src/ontology/mondo-edit.obo

```diff
@@ -346396,7 +346357,6 @@ xref: ICD10CM:I42.2 {source="Orphanet:324525/attributed", source="Orphanet:32452
 xref: Orphanet:324525 {source="MONDO:equivalentTo"}
 xref: UMLS:CN204073 {source="MONDO:equivalentTo"}
 is_a: MONDO:0016387 ! mitochondrial oxidative phosphorylation disorder
```

Comment:
Flagging since this is the only remaining `is_a` and it does not have a properly formatted source.

Reply:
I added my orcid